timetracker/entries/forms.py

```diff
@@ -1,23 +1,25 @@
 from django import forms
 from django.utils import timezone
 
-from .models import Project, Client
+from .models import Client, Entry, Project
 
 
-class ClientForm(forms.Form):
-    name = forms.CharField()
+class ClientForm(forms.ModelForm):
+    class Meta:
+        model = Client
+        fields = ('name',)
 
 
-class ProjectForm(forms.Form):
-	name = forms.CharField()
-	client = forms.ModelChoiceField(queryset=Client.objects.all())
+class ProjectForm(forms.ModelForm):
+    class Meta:
+        model = Project
+        fields = ('client','name')
 
 
-class EntryForm(forms.Form):
-    start = forms.DateTimeField(label="Start Time", help_text="Format: 2006-10-25 14:30")
```

Comment:
You're loosing the help text for the form fields here. Either define them as `help_texts` on `EntryForm.Meta` or as part of the respective model field.